pom.xml

```diff
@@ -739,6 +739,127 @@
+            <artifactId>httpcore</artifactId>
+            <version>4.4.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+            <version>2.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+            <version>2.5.6.SEC03</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-embedder</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.sisu</groupId>
+            <artifactId>org.eclipse.sisu.plexus</artifactId>
+            <version>0.3.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-aether-provider</artifactId>
+            <version>3.3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+            <artifactId>icon-set</artifactId>
+            <version>2.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-classworlds</artifactId>
+            <version>2.5.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-compat</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm</artifactId>
+            <version>5.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm-tree</artifactId>
+            <version>5.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>symbol-annotation</artifactId>
+            <version>1.10</version>
+        </dependency>
+        <dependency>
+            <groupId>com.infradna.tool</groupId>
+            <artifactId>bridge-method-annotation</artifactId>
+            <version>1.15</version>
+        </dependency>
+        <dependency>
+            <groupId>org.antlr</groupId>
+            <artifactId>antlr4-runtime</artifactId>
```

Comment:
Ditto I think.

Reply:
Hmm, in this case test dependency wire mock depends on antlr4-runtime 4.5.1-1 where as credentials on 4.5. So we need one version for test dependency and another for runtime, excluding antlr4-runtime from credentials won't work (I think) since wire mock is test dependency. In this case better to define antlr4-runtime explicit dependency from corresponding blueocean module blueocean-butbucket-pipeline?

```
Require upper bound dependencies error for org.antlr:antlr4-runtime:4.5 paths to dependency are:
+-io.jenkins.blueocean:blueocean-bitbucket-pipeline:1.3.0-beta-3-SNAPSHOT
  +-io.jenkins.blueocean:blueocean-pipeline-api-impl:1.3.0-beta-3-SNAPSHOT
    +-org.jenkins-ci.plugins:credentials:2.1.14
      +-org.antlr:antlr4-runtime:4.5
and
+-io.jenkins.blueocean:blueocean-bitbucket-pipeline:1.3.0-beta-3-SNAPSHOT
  +-com.github.tomakehurst:wiremock:2.6.0
    +-com.github.jknack:handlebars:4.0.6
      +-org.antlr:antlr4-runtime:4.5.1-1
```

Reply:
You could just exclude the dep from `credentials` or `blueocean-pipeline-api-impl` or `blueocean-bitbucket-pipeline`.